certified-connectors/Data247logic/README.md

```diff
@@ -0,0 +1,20 @@
+## Prerequisites

```

Comment:
@Data247logic Please provide a detailed description of your connector which can be helpful to the users who may want to contribute to your connector.

Reply:
already done, please review.